src/mocks/tests/common.test.ts

```diff
@@ -0,0 +1,32 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { generateNumber, sleep } from '../common';
+
+configure({ adapter: new Adapter() });
+
+describe('generateNumber', () => {
```

Comment:
Could you redo this (related to number of digits) with approach that @follan uses there:
https://github.com/cognitedata/gearbox.js/blob/master/src/components/TimeseriesChart/dataLoader.test.tsx#L84

Reply:
done :-)